### PR TITLE
Fix stylesheet option bug, allow local paths too

### LINKED ISSUE #10: Bug with stylesheet
When using the `-s` option, the stylesheet isn't linked correctly to the generated files.

For example, for the following file structure,
image

The user would execute:
```
ts-node .\src\index.ts .\examples\ -s .\examples\styles\stylesheet.css
```

This generates
image

which is the correct path from user's perspective, but since the files are generated in a different directory from the user, an adjusted path is expected.

In this example, should be
```
.\examples\styles\stylesheet.css
```

### PR DESCRIPTION
**Pull request by Amnish04**
Hi @avelynhc, sorry for the delay. But I just finished fixing the bug we talked about. Any css path the user enters from their context is now resolved to a valid relative path.

This closes #10

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -32,7 +33,16 @@ const argv = yargs
 
 let cssLink = '';
 if (argv.stylesheet !== '') {
-    cssLink = argv.stylesheet;
+    if (isURL(argv.stylesheet)) {
+        cssLink = argv.stylesheet;
+    }
+    else {
```
Comment (avelynhc) [inline src/index.ts:39]: please move the `else` statement with a space to line 38 to comply with the existing code format.

File: src/index.ts [hunk 2]
```diff
@@ -32,7 +33,16 @@ const argv = yargs
...
+        const ouptutDirectory = OUTPUT_DIR;
+        const filePath = argv.stylesheet;
+        const relativePath = path.relative(ouptutDirectory, filePath);
+
+        cssLink = relativePath;
```
Comment (avelynhc) [inline src/index.ts:44]: can you do line 40 - line 44 without variables? 

```typescript
cssLink = path.relative(OUTPUT_DIR, argv.stylesheet)
```

**Review by avelynhc — CHANGES_REQUESTED** [review]

**Comment by Amnish04** [comment]
@avelynhc Just pushed the requested changes

**Review by avelynhc — APPROVED** [review]
LGTM. Thank you!

**Comment by Amnish04** [comment]
@avelynhc I think you forgot to merge the PR

**Comment by avelynhc** [comment]
@Amnish04 Thanks for letting me know. It's merged now. Thanks for the contribution again!

**Pull request merged**